### DISCUSSION TITLE
WCAG 2.2 Open Discussion

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5569] [Accessibility]
It’s official! [WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/) success criteria are now part of the standards and guidelines recommended by the Web Accessibility Initiative working group! 

What does this mean for USWDS? Well, let's talk about it. 

What's interesting or challenging to you about WCAG 2.2? 
Do you see the new success criteria affecting current components and patterns?
What kind of guidance would you like to see from us? 

We expect to work to support WCAG 2.2 AA, and it seems like our most important work may lie with reliable, clear focus states. 

Let's use this space for an open discussion about what WCAG 2.2 might mean for the design system and for you. Let us know what's on your mind and help us work through what we'll need to do to support your team.

**Comment by bruce-usab** [comment]
Please excuse this shameless plug for [An overview of new web accessibility requirements from WCAG 2.2 and 2.1](https://www.section508.gov/iaaf/archives/agenda-2023/#day3breakout2) session at [IAAF](https://www.section508.gov/iaaf/).  We will include mention of [M-23-22](https://www.whitehouse.gov/omb/management/ofcio/delivering-a-digital-first-public-experience/) and this sentence:  
> In addition to the accessibility standards required by Section 508, agencies should apply the most current Web Content Accessibility Guidelines (WCAG) published by the World Wide Web Consortium (W3C) to websites and web applications, where possible.

**Comment by mgifford** [comment]
This is totally the right evergreen language that agencies should embrace. Thanks @bruce-usab 

Worth noting that Accessibility Insights supports 2.2 now
https://devblogs.microsoft.com/engineering-at-microsoft/accessibility-insights-now-supports-wcag-2-2-aa/

as does axe core:
https://www.deque.com/blog/axe-core-4-5-first-wcag-2-2-support-and-more/

So there's really no excuse.

**Comment by mgifford** [comment]
Worth noting that the UK's GDS just released this https://accessibility.blog.gov.uk/2024/03/13/wcag-2-2-detective-skills/
